editMember.php

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- CSS & Js Sheets -->
+    <link href="styles/styles.css" rel="stylesheet" type="text/css" />
+    <script type="text/javascript" src="scripts/searchmember.js"></script>
+    <link rel="icon" href="./assets/icons/SiOverleaf.svg" type="image/icon" />
+    <title>GoToGro | Edit Member</title>
+</head>
+
+<body>
+    <?php
+    include_once("navbar.inc");
+    ?>
+    <div class="editmember">
+        <h1 class="white-text">Search for member </h1>
+        <form method="post" action="searchMEMBER.php" novalidate="novalidate">
+            <div class="inputField">
+                <!-- <label for="member_id">Search for member </label>-->
```

Comment:
Was this for comparing the heading as a label vs a h1?? I'm not sure what the process is for doing design changes, might be a Khang question.